src/lib/y2storage/partition.rb

```diff
@@ -126,6 +126,30 @@ def disk
       partitionable.is_a?(Disk) ? partitionable : nil
     end
 
+    # Whether the first sector of the partition is aligned according to
+    # the partition table grain.
+    #
+    # @return [Boolean]
+    def start_aligned?
+      grain = partition_table.align_grain
+      block_size = region.block_size
+      sector = region.start
+      overhead = (block_size * sector) % grain
+      overhead.zero?
+    end
+
+    # Whether the final sector of the partition is aligned according to
+    # the partition table grain.
+    #
+    # @return [Boolean]
+    def end_aligned?
+      grain = partition_table.align_grain
+      block_size = region.block_size
+      sector = region.end
+      overhead = (block_size * sector + block_size) % grain
+      overhead.zero?
```

Comment:
I'm surprised that none of our cops complains about code duplication in `start_aligned?` end `end_aligned?`. Just saying :wink:

BTW, is `start_aligned?` really used of it's just defined for symmetry reasons?

Reply:
`start_aligned?` is only used in tests.